app/dose-event-card/dose-event-card.component.html

```diff
@@ -17,7 +17,7 @@
 
     <ion-card-content>
         <p>
-            {{doseEvent.user.name}} has taken all his medication for the day.
+            {{doseEvent.user.name}} has taken all their medication for the day.
```

Comment:
Though "his" is the masculine and gender-neutral pronoun, nobody uses it that way anymore.

Though "their" is the plural and technically incorrect, it is common in informal speech and writing.

We are Dose Amigos.  We can be informal.